### DISCUSSION TITLE
Can't add more than one rig to export settings for mesh

### DISCUSSION THREAD

**Discussion by VABG** [WolvenKit/WolvenKit#507] [Bug Report]
When trying to add multiple rigs for exporting of a rigged/skinned character mesh, any rig after the first is discarded, as can be seen by going back into the mesh export settings 

Mesh tested is the tank top of the character Sasquatch from Cyberpunk 2077 (among other meshes).
" t1_003_wm_tank__animal_boss.mesh" tested with "man_massive" and its variations, and "animal_boss" variations. As none worked, seemingly requires multiple rigs?

**Example Error Message on Export**
"2021-06-29 19:16:33 [17] Bone: l_butterfly_top_CRV_bot_out_JNT is not present in the Provided .rig(s).
Input .rig(s) are incompatible or incomplete, Please provide a/more compatible .rig(s)
TIP: 1. For body .rig(s) provide {BodyType}_base_deformations.rig instead of {BodyType}_base.rig.
2. if Input .mesh(s) contains any dangle/physics bones, provide the compatible dangle.rig also."

Based on the error message I think I can assume multi-rig mesh export is meant to be supported?

**How to reproduce**
1. Find a mesh that requires multiple rigs and try to export.
2. Get error message above (or similar).
3. Add more than one rig to the collection, press confirm.
4. Get same error message again when exporting.
5. Go back in and find that you only have one rig in the collection.

Or just try adding more than one rig to anything, confirm, go back in and find all but one gone.
On a brighter note, Sasquatch's upper body got exported just fine.

- OS: Windows 10
- Tested on WolvenKit 1.8.0.0 and hotfix 1.8.1.0

**Comment by Offline-R503B** [comment]
**Thanks for the bug report!**

We're going to look into the issue, and will come back to you as soon as possible with an update! 

When your issue is fixed we will tag the relevant "Pull Request"  in this topic.

- Offline

**Comment by kainotoa** [comment]
Yes multi-rig mesh export is supported, in the export options u have a "multimesh" option which is both for merging multiple meshes and multiple rigs together, you can use that.

**Reply by VABG** [reply]
Thanks! That seems to have worked, with a minor note.

If I don't add the original mesh (the selected mesh) to the multi-mesh export, it doesn't export anything, and provides no error messages, but reports it as a successful export.

Otherwise working great! Seems like this is mostly a UI/UX type of issue. More specifically the "rig(s)" text where the pluralization not applicable, and of course the above.
